xgboost_synthetic/util.py

```diff
@@ -11,16 +11,13 @@
 def notebook_setup():
   # Install the SDK
 
-  subprocess.check_call(["pip3", "install", "-r", "requirements.txt"])
-  subprocess.check_call(["pip3", "install", KFP_PACKAGE, "--upgrade"])
+  subprocess.check_call(["pip3", "install", "--user", ""-r", "requirements.txt"])
+  subprocess.check_call(["pip3", "install", "--user", KFP_PACKAGE, "--upgrade"])
 
   logging.basicConfig(format='%(message)s')
   logging.getLogger().setLevel(logging.INFO)
 
   subprocess.check_call(["gcloud", "auth", "configure-docker", "--quiet"])
-  subprocess.check_call(["gcloud", "auth", "activate-service-account",
-                         "--key-file=" + os.getenv("GOOGLE_APPLICATION_CREDENTIALS"),
-                         "--quiet"])
```

Comment:
you might want to keep this in place with a guard of:
if os.getenv('GOOGLE_APPLICATION_CREDENTIALS'):

so that the notebook works on a 0.6.x deployment as well.

Reply:
done